drag_and_drop_v2/utils.py

```diff
@@ -5,3 +5,24 @@
 # Make '_' a no-op so we can scrape strings
 def _(text):
     return text
+
+
+def ngettext(text_singular, text_plural, n):
+    if n == 1:
+        return text_singular
+    else:
+        return text_plural
+
+
+class FeedbackMessages(object):
+    FINAL_ATTEMPT_TPL = _('Final attempt was used, highest score is {score}')
+    MISPLACED_ITEMS_RETURNED = _('Misplaced items were returned to item bank.')
+
+    CORRECTLY_PLACED_SINGULAR_TPL = _('Correctly placed {correct_count} item.')
+    CORRECTLY_PLACED_PLURAL_TPL = _('Correctly placed {correct_count} items.')
```

Comment:
@e-kolpakov Sorry for not catching this earlier, but I think these need to be marked with `ngettext` for gettext utils to extract the strings into `po` files properly. Strings used inside `ngettext` are represented differently from regular `gettext` strings inside `po` files, so wrapping singular and plural strings into `_()` individually will not produce correct entries in `po` files.

I am not sure it is possible to represent strings used inside ngettext as constants due to problems with too early translations: http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/conventions/internationalization/i18n.html#translating-too-early